plugins/octant/cmd/traceflow-plugin/main.go

```diff
@@ -0,0 +1,409 @@
+			return nil
+		}
+
+		dstPod, err := request.Payload.String(dstPodCol)
+		if err != nil {
+			log.Printf("unable to get dstPod at string : %s", err)
+		}
+		if match := a.regExpMatch(podStrPattern, dstPod); !match {
+			return nil
+		}
+
+		// TODO: "Destination is a service" is not supported now.
+		/*	dstService, err := request.Payload.String(dstSvcCol)
+			if err != nil {
+				log.Printf("unable to get dstService at string : %s", err)
+			}*/
+
+		dstPort, err := request.Payload.String(dstPortCol)
+		if err != nil {
+			log.Printf("unable to get dstPort at string : %s", err)
+		}
+
+		protocol, err := request.Payload.String(protocolCol)
+		if err != nil {
+			log.Printf("unable to get dstPod at string : %s", err)
+		}
+		protocol = strings.ToUpper(protocol)
+
+		// Judge whether the name of trace flow is duplicated.
+		// If it is, then the user creates more than one traceflows in one second, which is not allowed.
+		tfName := srcPod + "-" + dstPod + "-" + time.Now().Format(TIME_FORMAT_YYYYMMDD_HHMMSS)
+		ctx := context.Background()
+		tfOld, err := a.client.OpsV1alpha1().Traceflows().Get(ctx, tfName, metav1.GetOptions{})
+		if err != nil {
+			log.Printf("unable to get traceflow \"%s\", detailed error: %s", tfName, err)
+		}
+		if tfOld.Name == tfName {
+			log.Printf("duplicate traceflow \"%s\": %+v. ", tfName, tfOld)
+			return nil
+		}
+
+		tf := &opsv1alpha1.Traceflow{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: tfName,
+			},
+			Spec: opsv1alpha1.TraceflowSpec{
+				Source: opsv1alpha1.Source{
+					Namespace: srcNamespace,
+					Pod:       srcPod,
+				},
+				Destination: opsv1alpha1.Destination{
+					Namespace: dstNamespace,
+					Pod:       dstPod,
+					//				Service:   dstService,
+				},
+				Packet: opsv1alpha1.Packet{
+					IPHeader: opsv1alpha1.IPHeader{
+						Protocol: supportedProtocols[protocol],
+					},
+				},
+			},
+		}
+		var sport, dport int
+		if srcPort != "" {
+			sport, err = strconv.Atoi(srcPort)
+			if err != nil {
+				log.Printf("failed to get source port: %s", err)
+				return nil
+			}
+		}
+		if dstPort != "" {
+			dport, err = strconv.Atoi(dstPort)
+			if err != nil {
+				log.Printf("failed to get destination port: %s", err)
+				return nil
+			}
+		}
+		switch tf.Spec.Packet.IPHeader.Protocol {
+		case TCPProtocol:
+			{
+				tf.Spec.Packet.TransportHeader.TCP = &opsv1alpha1.TCPHeader{
+					SrcPort: int32(sport),
+					DstPort: int32(dport),
+				}
+			}
+		case UDPProtocol:
+			{
+				tf.Spec.Packet.TransportHeader.UDP = &opsv1alpha1.UDPHeader{
+					SrcPort: int32(sport),
+					DstPort: int32(dport),
+				}
+			}
+		case ICMPProtocol:
+			{
+				tf.Spec.Packet.TransportHeader.ICMP = &opsv1alpha1.ICMPEchoRequestHeader{
+					ID:       0,
+					Sequence: 0,
+				}
+			}
+		}
+		tf, err = a.client.OpsV1alpha1().Traceflows().Create(ctx, tf, metav1.CreateOptions{})
+		if err != nil {
+			log.Printf("Failed to create traceflow CRD \"%s\", err: %s", tfName, err)
```

Comment:
"Failed" - all other log messages start with lower case.

Reply:
Logs are unified to begin with lower case letters.

Reply:
In Antrea convention, logs start from capital letter. Any reason Octant plugin uses a different way?

Reply:
My bad. Changed all logs to make them start with capital letters.